src/codeu/chat/ServerMain.java

```diff
@@ -53,8 +54,10 @@ public static void main(String[] args) {
 
     final int myPort = Integer.parseInt(args[2]);
 
-    final RemoteAddress relayAddress = args.length > 3 ?
-                                       RemoteAddress.parse(args[3]) :
+    final String dbPath = args[3];
```

Comment:
Since you're changing the command line args which will be accepted, it will be good to add comments describing the positional args and what they are for. I know there is no existing comments describing it, but it's a good practice to add these.